api/namex/models/name.py

```diff
@@ -71,8 +73,8 @@ def delete_from_db(self):
 class NameSchema(ma.ModelSchema):
     class Meta:
         model = Name
-        fields = ('name', 'state', 'choice', 'designation', 'consumptionDate', 'conflict1', 'conflict2', 'conflict3',
-                  'conflict1_num', 'conflict2_num', 'conflict3_num', 'decision_text')
+        fields = ('name', 'state', 'choice', 'designation', 'consumptionDate', 'corpNum', 'conflict1', 'conflict2',
+                  'conflict3', 'conflict1_num', 'conflict2_num', 'conflict3_num', 'decision_text')
```

Comment:
Having real consumptionDate data going to/from the front-end now causes validation errors when unrelated data is sent back to the API from the front-end. To prevent this, we need to remove consumptionDate from the json data (as_dict()). It's not used anywhere in the front-end yet so this is probably the best way to prevent bugs until the front-end can be managed.

Similarly, we also need to remove corpNum from as_dict() and NamesSchema. It will get overwritten during saves if it remains inthe NamesSchema. 

That means a few tests will also have to be changed to remove that data from the schema, until we can put it back.